README.rst

```diff
@@ -64,31 +64,41 @@ To cite a specific version of Cobrawap please see version-specific DOIs at:
 
 To cite Cobrawap, please use:
 
-Gutzen, R., De Bonis, G., De Luca, C., Pastorelli, E., Capone, C., Allegra Mascaro, A. L., Resta, F., Manasanch, A., Pavone, F. S., Sanchez-Vives, M. V., Mattia, M., Grün, S., Paolucci, P. S., & Denker, M. (2022). *Comparing apples to apples—Using a modular and adaptable analysis pipeline to compare slow cerebral rhythms across heterogeneous datasets*. arXiv:2211.08527. `https://doi.org/10.48550/arXiv.2211.08527 <https://doi.org/10.48550/arXiv.2211.08527>`_
+Robin Gutzen, Giulia De Bonis, Chiara De Luca, Elena Pastorelli, Cristiano Capone, Anna Letizia Allegra Mascaro, Francesco Resta, Arnau Manasanch, Francesco Saverio Pavone, Maria V. Sanchez-Vives, Maurizio Mattia, Sonja Grün, Pier Stanislao Paolucci, Michael Denker (2024), *A modular and adaptable analysis pipeline to compare slow cerebral rhythms across heterogeneous datasets*, Cell Reports Methods,
+Volume 4, Issue 1, `https://doi.org/10.1016/j.crmeth.2023.100681 <https://doi.org/10.1016/j.crmeth.2023.100681>`_
 
 
 License
 =======
 Cobrawap is open-source software and is licensed under the `GNU General Public License v3 <https://github.com/NeuralEnsemble/cobrawap/blob/master/LICENSE>`_.
 
 
-Involved partners
+The Cobrawap Community
 =================
-Cobrawap was originally developed in the context the `Human Brain Project <https://www.humanbrainproject.eu>`_ and is part of the `EBRAINS <https://www.ebrains.eu>`_ infrastructure. Partners involved in the collaboration and use case within the project (*WaveScalES*) are:
+Cobrawap is currently provided as a `tool <https://www.ebrains.eu/tools/cobrawap>`_ of the `EBRAINS <https://www.ebrains.eu>`_ infrastructure and included in the `EBRAINS-Italy <https://www.ebrains-italy.eu/>`_ initiative. Further details on fundings and resources are in the `Acknowledgments <https://github.com/NeuralEnsemble/cobrawap/blob/master/doc/source/acknowledgments.rst>`_ file in the doc session.
```

Comment:
At the very end of the statement, there is a typo: "session" -> "section", or even better "folder".

Reply:
done. new commit added.